integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -941,4 +943,40 @@ object CommonUtil {
     }
   }
 
+  def setTempStoreLocation(
```

Comment:
Suggesting to add a parameter to use a different temp folder for different operation.
for data loading, carbon_**dataload**\_\<nanotime\>\_\<split index\>
for compaction, carbon_**compact**\_\<nanotime\>\_\<split index\>
for partition, carbon_**partition**\_\<nanotime\>\_\<split index\>
for streaming handoff, carbon_**handoff**\_\<nanotime\>\_\<split index\>

Reply:
this is for temporary folder. no need to add the operation to the folder name.Moreover we are using nanotime. which will be different across operations and would not collide